src/navigator/main-stack.tsx

```diff
@@ -89,6 +91,8 @@ export const MainStackScreen = () => {
     dispatch(loadTezosBalanceActions.submit());
     dispatch(loadTokensActions.submit());
     dispatch(loadSelectedBakerActions.submit());
+    dispatch(loadMarketTopCoinsActions.submit());
+    dispatch(loadMarketCoinsSlugsActions.submit());
```

Comment:
maybe let's do this on the `Market` screen? this will reduce the initial load on the app